cfgov/cfgov/urls.py

```diff
@@ -232,9 +232,6 @@
         namespace='transcripts')),
 ]
 
-# TODO: Remove prototype landing page routes when all organisms and molecules have been implemented elsewhere.
-if settings.DEBUG :
-    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
those urls have been removed but we're still going to need this particular pattern for images.
```
if settings.DEBUG :		
    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)		
``